packages/nodes-base/nodes/Kafka/KafkaTrigger.node.ts

```diff
@@ -147,6 +178,15 @@ export class KafkaTrigger implements INodeType {
 
 		const options = this.getNodeParameter('options', {}) as IDataObject;
 
+		const useSchemaRegistry = this.getNodeParameter('useSchemaRegistry', 0) as boolean;
+
+		const schemaRegistryUrl = this.getNodeParameter('schemaRegistryUrl', 0) as string;
+
+		let registry: SchemaRegistry;
+		if (useSchemaRegistry) {
+			registry = new SchemaRegistry({ host: schemaRegistryUrl });
```

Comment:
you can move this line where you will use the registry and you can switch `let` with `const` and make the assignment in the same line.

Reply:
done